src/core/server/plugins/discovery/plugin_manifest_parser.test.ts

```diff
@@ -139,7 +138,7 @@ test('return error when plugin version is missing', async () => {
   });
 });
 
-test('return error when plugin expected OpenSearch Dashboards version is lower than actual version', async () => {
+test('return error when plugin expected OpenSearch Dashboards version has different major version', async () => {
```

Comment:
Why change these unit tests? They should remain the same as the default behavior hasn't been changed, am I right?